packages/backend/src/overrideUserPassword.ts

```diff
@@ -6,35 +6,42 @@ import Logger from './logging/logger';
 import { UserModel } from './models/UserModel';
 
 (async function init() {
-    Logger.warn(`Override user password`);
-    const email: string | undefined = process.argv[2];
-    const newPassword: string | undefined = process.argv[3];
+    try {
+        Logger.warn(`Override user password`);
```

Comment:
I didn't know about this script. is this documented somewhere ?

Reply:
In our public docs. 
Useful script for self hosted instances without a SMTP. So admins can reset passwords for users.
https://docs.lightdash.com/self-host/customize-deployment/use-sso-login-for-self-hosted-lightdash#passwords